This patch is required for FreeBSD 2.0 because password crypt vs cipher
technology has now been split into different libraries.  In this case,
USE_CRYPT really means "use cipher."

*** Makefile	Thu Oct 27 14:12:41 1994
--- Makefile	Thu Oct 27 14:14:46 1994
***************
*** 76,84 ****
  .endif
  
  .if defined(USE_CRYPT)
! NETBSD_LIBS		= -lcrypt -lutil
  .else
! NETBSD_LIBS		= -lutil
  .endif
  CMUCS_LIBS		= -lsys
  OSF_LIBS		= -lbsd
--- 76,84 ----
  .endif
  
  .if defined(USE_CRYPT)
! NETBSD_LIBS		= -lcipher -lcrypt -lutil
  .else
! NETBSD_LIBS		= -lcrypt -lutil
  .endif
  CMUCS_LIBS		= -lsys
  OSF_LIBS		= -lbsd
